app/views/onboarding/Onboarding1.js

```diff
@@ -16,11 +16,11 @@ import languages, {
   setUserLocaleOverride,
   supportedDeviceLanguageOrEnglish,
 } from './../../locales/languages';
-import ButtonWrapper from '../../components/ButtonWrapper';
+import EulaModal from '../../components/EulaModal';
```

Comment:
don't use default exports, IDE autocomplete is much better at finding named exports (VSCode, in this case, but probably others)